<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calc</class>
 <widget class="QDialog" name="calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QPushButton" name="value_0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="add">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QPushButton" name="value_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="sub">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="value_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="value_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="value_9">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="value_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="value_7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="div">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="value_6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QPushButton" name="value_5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="mul">
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLCDNumber" name="calc_display">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="value_8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="clear_p">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="equals">
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="clear_a">
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>value_1</tabstop>
  <tabstop>value_2</tabstop>
  <tabstop>value_3</tabstop>
  <tabstop>value_4</tabstop>
  <tabstop>value_5</tabstop>
  <tabstop>value_6</tabstop>
  <tabstop>value_7</tabstop>
  <tabstop>value_8</tabstop>
  <tabstop>value_9</tabstop>
  <tabstop>value_0</tabstop>
  <tabstop>equals</tabstop>
  <tabstop>add</tabstop>
  <tabstop>sub</tabstop>
  <tabstop>mul</tabstop>
  <tabstop>div</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
